scripts/analysisTools/w_mass_13TeV/makeWMCefficiency3D.py

```diff
@@ -260,7 +260,7 @@ def getEtaPtEff(n_pass, n_tot, getRoot=False, rootName="", rootTitle=""):
     if len(postfix):
         postfix = "_" + postfix
 
-    resultDict.update({"meta_info" : output_tools.metaInfoDict(args=args)})    
+    resultDict.update({"meta_info" : narf.ioutils.make_meta_info_dict(args=args, wd=common.wremnants_dir)})    
```

Comment:
Is this actually the desired directory?

It should point to the top directory of the repository, rather than the "wremnants" subdirectory.

Reply:
Indeed, this was not intended, it should be corrected now.